source/linux/Foundational_Components/Kernel/Kernel_Drivers/Network/CPSW-Ethernet.rst

```diff
@@ -40,14 +40,25 @@ of the CPSW Ethernet Subsystem:
 
  For further details regarding the CPSWng instance, refer :ref:`cpswng`
 
-.. ifconfig:: CONFIG_part_family in ('AM62X_family','AM62AX_family','AM64X_family','AM62PX_family','AM62LX_family')
+.. ifconfig:: CONFIG_part_family in ('AM62X_family','AM62AX_family','AM64X_family','AM62LX_family')
 
    CPSW3g
 
  The CPSW3g instance is a part of the Main domain and is controlled by the am65-cpsw-nuss
  driver in Linux.
  For further details regarding the CPSW3g instance, refer :ref:`cpsw3g`
 
+.. ifconfig:: CONFIG_part_family in ('AM62PX_family')
+
+   CPSW3g
+
+ The CPSW3g instance can be configured to operate in two ways:
+   #. CPSW3g Virtual Client
+   #. CPSW3g Native Ethernet
+
+ To generalize, CPSW3g instance can be referred to as CPSWng with 'n' being '3'.
+ For further details regarding the CPSW3g instance, refer :ref:`cpswng`
+
```

Comment:
Can you normalize the indentation in these sections? It's not causing render issues as it's currently being treated as a block quote instead of a paragraph, but it is harder to follow with this mixed indentation depth.